addons/spottingscope/CfgWeapons.hpp

```diff
@@ -27,7 +27,7 @@ class CfgWeapons {
             reticleDetailTextures[] = {
                 // start at > this magnification, reticleTexture, reticleTextureSize, reticleTextureNight (optional)
                 {0,  QPATHTOF(data\ace_spottingscope_reticle_b1_ca.paa), 1,   QPATHTOF(data\ace_spottingscope_reticle_b1_ca.paa)},
-                {9,  QPATHTOF(data\ace_spottingscope_reticle_b2_ca.paa), "1/2", QPATHTOF(data\ace_spottingscope_reticle_b2_ca.paa)},
+                {9,  QPATHTOF(data\ace_spottingscope_reticle_b2_ca.paa), 1/2, QPATHTOF(data\ace_spottingscope_reticle_b2_ca.paa)},
                 {14, QPATHTOF(data\ace_spottingscope_reticle_b3_ca.paa), "1/3", QPATHTOF(data\ace_spottingscope_reticle_b3_ca.paa)},
                 {19, QPATHTOF(data\ace_spottingscope_reticle_b4_ca.paa), "1/4", QPATHTOF(data\ace_spottingscope_reticle_b4_ca.paa)},
```

Comment:
lint might only be going 1 level deep on arrays I will take a look

Reply:
Yeah this was a global find and replace for 1/2 since it was common, HEMTT never flagged the others